### PR TITLE
Add FUJIFILM X RAW STUDIO.app v1.9.0

### PR DESCRIPTION
**Pull request by timestride**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

### CODE REVIEW

File: Casks/fujifilm-x-raw-studio.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  depends_on macos: '>= :sierra'
+
+  app 'FUJIFILM X RAW STUDIO.app'
+
+  uninstall quit: 'com.fujifilm.denji.X-RAW-STUDIO'
+
+  zap trash: [
+               '~/Library/Application Support/com.fujifilm.denji',
+               '~/Library/Preferences/com.fujifilm.denji.X-RAW-STUDIO.plist',
+               '~/Library/Saved Application State/com.fujifilm.denji.X-RAW-STUDIO.savedState/',
```
Comment (vladimyr) [inline Casks/fujifilm-x-raw-studio.rb:18]:
```suggestion
               '~/Library/Saved Application State/com.fujifilm.denji.X-RAW-STUDIO.savedState',
```

**Review by vladimyr — APPROVED** [review]

**Comment by vitorgalvao** [comment]
Needs to be submitted to [homebrew/cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers). We [define drivers](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) as:

> Software to make a hardware peripheral recognisable and usable by the system. If the software is useless without the peripheral, it’s considered a driver.

Note that repo has [special naming rules](https://github.com/Homebrew/homebrew-cask-drivers/blob/master/CONTRIBUTING.md#naming-driver-casks).

**Comment by timestride** [comment]
Oh man, so close but so far away. Thanks for reviewing and letting me know.

**Comment by vitorgalvao** [comment]
> Oh man, so close but so far away. Thanks for reviewing and letting me know.

It’s just a case of submitting to the correct repo and I’ll merge it. But please add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md): `https://fujifilm-x.com/en-us/support/download/software/x-raw-studio/`

**Comment by timestride** [comment]
Submitted https://github.com/Homebrew/homebrew-cask-drivers/pull/1557

**Pull request closed without merging**